Guide.docc/Swift6Mode.md

```diff
@@ -0,0 +1,72 @@
+# Enabling The Swift 6 Language Mode
+
+Guarantee your code is free of data races by enabling the Swift 6 language mode.
+
+## Using the Swift compiler
+
+To enable complete concurrency checking when running `swift` or `swiftc`
+directly at the command line, pass `-swift-version 6`:
+
+```
+~ swift -swift-version 6 main.swift
+```
+
+## Using SwiftPM
+
+### Command-line invocation
+
+`-swift-version 6` can be passed in a Swift package manager command-line
+invocation using the `-Xswiftc` flag:
+
+```
+~ swift build -Xswiftc -swift-version 6
+~ swift test -Xswiftc -swift-version 6
+```
+
+### Package manifest
+
+To change the language mode for package as a whole:
```

Comment:
Ah sorry, I didn't see this until you closed the corresponding issue. `swift-tools-version: 6.0` implies language mode 6, so the `swiftLanguageVersions` isn't needed in that case. We probably want to explain that and then show the opposite, ie. bump to tools version 6 and then change language mode on a per-target basis to 5 in order to migrate (or set to 5 in the top level if you're not ready to migrate).

Reply:
yeahh, you are 100% right.